maven-plugin/src/main/resources/settings.xml

```diff
@@ -24,9 +24,17 @@
             </properties>
             <repositories>
                 <repository>
-                    <id>openmrs-repo</id>
+                    <id>archetype</id>
                     <name>OpenMRS Nexus Repository</name>
                     <url>http://mavenrepo.openmrs.org/nexus/content/repositories/public</url>
+                     <releases>
+				     <enabled>true</enabled>
+					 <checksumPolicy>fail</checksumPolicy>
+					 </releases>
+					 <snapshots>
+					 <enabled>true</enabled>
+					 <checksumPolicy>warn</checksumPolicy>
+					 </snapshots>
```

Comment:
Can you please ensure to consistently only use spaces for the formatting here? Also, we shouldn't remove the `openmrs-repo` identifier nor add the policies to it. We should instead create a new repository entry, even though it's referring to the same repository. This allows us to have a specific policy that applies to downloading archetypes, but not necessarily to other Maven artefacts.